common/network-tuning.sh

```diff
@@ -1,9 +1,9 @@
 #!/bin/bash
 set -ex
 
-# Place the topology file in /opt/microsoft
-sudo mkdir -p /opt/microsoft
-sudo bash -c "cat > /opt/microsoft/ndv4-topo.xml" <<'EOF'
+# Place the NDv4 topology file in /opt/microsoft/ndv4
```

Comment:
I would recommend keeping the existing file. Some customers may have run scripts with NCCL topo file hardcoded.

Reply:
Reverted